roborazzi-compose-preview-scanner-support/src/main/java/com/github/takahirom/roborazzi/RoborazziPreviewScannerSupport.kt

```diff
@@ -18,7 +20,9 @@ fun ComposablePreview<AndroidPreviewInfo>.captureRoboImage(
   if (!roborazziOptions.taskType.isEnabled()) return
   val composablePreview = this
   captureRoboImage(filePath, roborazziOptions, configBuilder) {
-    composablePreview()
+    CompositionLocalProvider(LocalInspectionMode provides true) {
+      composablePreview()
+    }
   }
```

Comment:
When running the Preview Screenshot Test, it is normally expected to produce the same result as during the Preview rendering, so there is no option to choose between true or false. If there is a need to make it selectable, I will modify it accordingly.

Reply:
We could have something like `roborazz.compose.inspectionmode = true`. The default would be null. However, this might be a little confusing because we can't inject the setting if you use your activity.

Reply:
Then better to leave it as ComposeOption